.circleci/config.yml

```diff
@@ -166,21 +161,42 @@ jobs:
     <<: *executor
     steps:
       - *attach_workspace
-      - *import_image
+      - *import_final_image
+      - checkout
+      - *install_pytest
       - run: |
           mkdir ~/testresults
-          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ~/testresults:/output signalfx-agent-dev pytest -n8 --junitxml=/output/integration_tests.xml -m "not packaging and not installer and not k8s" tests
+          mkdir -p /tmp/scratch
+          mkdir bundle
+          docker run --rm -v `pwd`/bundle:/tmp/bundle quay.io/signalfx/signalfx-agent-dev:latest \
+              bash -c 'cp -r /bin /etc /jvm /lib /lib64 /neopy /plugins /run /var /tmp/bundle/'
```

Comment:
I'm wary of specifying so many directories because this is going to break unexpectedly if the dir layout of the image changes.  Here is a more robust way of doing this:
```
  mkdir bundle
  cid=$(docker create quay.io/signalfx/signalfx-agent-dev:latest true)
  docker export $cid | tar -C $(pwd)/bundle/ -xf -
  docker rm -fv cid
```

This requires no scratch dir and is robust against future changes to the image layout.